zerver/lib/integrations.py

```diff
@@ -94,6 +96,40 @@ def add_doc_context(self, context):
         # type: (Dict[Any, Any]) -> None
         self.doc_context = context
 
+class BotIntegration(Integration):
+    DEFAULT_LOGO_STATIC_PATH_PNG = 'static/images/bots/{name}.png'
+    DEFAULT_LOGO_STATIC_PATH_SVG = 'static/images/bots/{name}.svg'
```

Comment:
Agreed that for the short term, that's fine.
For longer term, I think it would be great to have a library of 5-10 generic bot images, that each come in 10 random different colors, or something. They are only 30x30 pixels or so (I think), so seems doable?